### ISSUE TITLE
Resources drawer - Sole use of color to indicate selected tab

### ISSUE THREAD

**Issue by swashbuck** [adaptlearning/adapt-contrib-vanilla#337]
In the resources drawer, the tablist of options conveys selection only through a change in background color. Some users with low vision or color blindness may have difficulty perceiving this difference.

 Note: Some tab options have a slightly higher top border, but this only appears on the top row of tabs.

**Steps to Reproduce:**

1. Activate the Resources button in the header.
2. Activate the "Resources" option in the side drawer, then note the presentation of the selected resource tab with non-selected options. 

**Actual Behavior:**

Selected options are presented with a black background while non-selected options are presented with a dark gray background.

**Expected Behavior:**

Provide a non-color dependent indicator of selection to ensure all users can consistently identify the active tab in the grouping.
